Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/SDK/Maya/Scripts/Python/legacy_asset_converter/main.py

```diff
@@ -78,6 +78,8 @@
 
 module_name = 'legacy_asset_converter.main'
 log_file_path = os.path.join(settings.DCCSI_LOG_PATH, f'{module_name}.log')
+if not os.path.exists(settings.DCCSI_LOG_PATH):
+    os.makedirs(settings.DCCSI_LOG_PATH)
```

Comment:
nit: you can use the `exist_ok` argument as well
```suggestion
os.makedirs(settings.DCCSI_LOG_PATH, exist_ok=True)
```